fsspec/implementations/local.py

```diff
@@ -52,6 +52,8 @@ def rmdir(self, path):
 
     def ls(self, path, detail=False, **kwargs):
         path = self._strip_protocol(path)
+        # root_marker if stripped path return an empty string
+        path = path or self.root_marker
```

Comment:
Do you think this should live in `_strip_protocol`, like 
```
        return make_path_posix(path).rstrip("/") or cls.root_marker
```
?

Reply:
Indeed, this specific issue is linked to `_strip_protocol`, not to `ls`